packages/aws-lambda/test/multiple_data/test.js

```diff
@@ -152,6 +160,87 @@ describe('multiple data lambda handler', function () {
     });
   });
 
+  describe('[batching enabled] with 100 iterations', function () {
+    this.timeout(config.getTestTimeout() * 2);
+    let control;
+
+    before(async () => {
+      control = new Control({
+        faasRuntimePath: path.join(__dirname, '../runtime_mock'),
+        handlerDefinitionPath: path.join(__dirname, './lambda'),
+        startBackend: true,
+        env: {
+          INSTANA_AGENT_KEY: instanaAgentKey,
+          WITH_CONFIG: 'true',
+          INSTANA_NUMBER_OF_ITERATIONS: 100,
```

Comment:
Suggestion: We can consider adding this env variables in our public doc.

Reply:
The comment is on the wrong line I think?

INSTANA_SPANBATCHING_ENABLED is documented
https://www.ibm.com/docs/en/instana-observability/current?topic=nodejs-collector-configuration#automatic-aggregation-of-short-exit-calls

But the docs are not so good.

I think we need to re-work the feature first and then document only whats needed.
Because documenting INSTANA_DEV_MIN_DELAY_BEFORE_SENDING_SPANS is not right.